### PR TITLE
IGNITE-15737 IoStatisticsSelfTest#testPersistentIOGlobalStat failure.

### PR DESCRIPTION
**Pull request by tkalkirill**
https://issues.apache.org/jira/browse/IGNITE-15737

### CODE REVIEW

File: modules/core/src/test/java/org/apache/ignite/internal/metric/IoStatisticsSelfTest.java [hunk 1]
```diff
@@ -145,28 +151,26 @@ private void ioStatGlobalPageTrackTest(boolean isPersistent) throws Exception {
         long physicalReadsCnt = physicalReads(mmgr, CACHE_GROUP, DEFAULT_CACHE_NAME, null);
 
         if (isPersistent)
-            Assert.assertTrue(physicalReadsCnt > 0);
+            Assert.assertThat(physicalReadsCnt, Matchers.greaterThan(0L));
```
Comment (rpuch) [inline modules/core/src/test/java/org/apache/ignite/internal/metric/IoStatisticsSelfTest.java:154]: If we import `assertThat` and `greaterThan` statically, then the assertion reads as a plain text (which is probably good for readability): 'assert that phisicalReadsCnt greater than 0`

File: modules/core/src/test/java/org/apache/ignite/internal/metric/IoStatisticsSelfTest.java [hunk 2]
```diff
@@ -190,9 +194,9 @@ private IgniteConfiguration getConfiguration(boolean isPersist) throws Exception
             DataStorageConfiguration dsCfg = new DataStorageConfiguration()
                 .setDefaultDataRegionConfiguration(
                     new DataRegionConfiguration()
-                        .setMaxSize(30L * 1024 * 1024)
-                        .setPersistenceEnabled(true))
-                .setWalMode(WALMode.LOG_ONLY);
+                        .setMaxSize(25 * MB)
```
Comment (rpuch) [inline modules/core/src/test/java/org/apache/ignite/internal/metric/IoStatisticsSelfTest.java:197]: Looks like this is THE change that fixes test flakiness. Looks like 30Mb produces a flaky test, but 25Mb does not. Could you please add a comment explaining why this value of 25Mb was chosen (is there a calculation that demonstrates that a test will be stable with 25Mb, or it's empirically chosen)?

File: modules/core/src/test/java/org/apache/ignite/internal/metric/IoStatisticsSelfTest.java (hunk 1 above)
Comment (tkalkirill) [reply modules/core/src/test/java/org/apache/ignite/internal/metric/IoStatisticsSelfTest.java:154]: Fix it

File: modules/core/src/test/java/org/apache/ignite/internal/metric/IoStatisticsSelfTest.java (hunk 2 above)
Comment (tkalkirill) [reply modules/core/src/test/java/org/apache/ignite/internal/metric/IoStatisticsSelfTest.java:197]: Tried to fix it

**Review by rpuch — APPROVED** [review]

**Pull request merged**